### PR TITLE
optimize SOA image size and doc corrections

### PR DESCRIPTION
**Pull request by sbattagi**
This PR contains the SOA Dockerfile fixes to optimize the image size.

Signed-off-by: Sambasiva Battagiri <sambasiva.battagiri@oracle.com>

### CODE REVIEW

**Review by vivek-sam — DISMISSED** [review]
Reviewed the code and it is in line with the internal Repo.

**Review by rdas0405 — APPROVED** [review]

**Comment by sbattagi** [comment]
Kindly merge this transaction.

**Comment by sbattagi** [comment]
@rjeberhard   Could you please merge this transaction? This fix is required for latest SOA release.

**Review by rjeberhard — CHANGES_REQUESTED** [review]

File: OracleSOASuite/dockerfiles/12.2.1.4/Dockerfile
Comment (Djelibeybi) [reply OracleSOASuite/dockerfiles/12.2.1.4/Dockerfile:100]: Agreed, please combine these `RUN` directives into a single one.

**Review by Djelibeybi — CHANGES_REQUESTED** [review]

File: OracleSOASuite/dockerfiles/12.2.1.4/Dockerfile
Comment (sbattagi) [reply OracleSOASuite/dockerfiles/12.2.1.4/Dockerfile]: Merged the run commands and committed the changes. Please review.

**Review by rjeberhard — APPROVED** [review]

**Review by Djelibeybi — APPROVED** [review]

**Comment by Djelibeybi** [comment]
Still waiting on approval from @rdas0405 whos is the code owner for this.

**Review by rdas0405 — APPROVED** [review]

**Pull request merged**
